Samples~/LootLockerExamples/8 - Virtual Currency Purchases/LootLockerVirtualStore.cs

```diff
@@ -0,0 +1,126 @@
+using UnityEngine;
+using UnityEngine.UI;
+using LootLocker.Requests;
+
+namespace LootLocker
+{
+    public class LootLockerVirtualStore : MonoBehaviour
+    {
+        public static LootLockerVirtualStore Instance { get; private set; }
+
+        public Transform storeUI;
+        public Transform virtualPurchaseProductPrefab;
+
+        public Text goldAmount;
+
+        public string catalogKey = "";
+        public string walletID = "";
+
+        private string currencyCode;
+
+        public void Awake()
+        {
+
+            if (Instance == null)
+            {
+                Instance = this;
+            }
+            else Destroy(gameObject);
+
+            LootLockerSettingsOverrider.OverrideSettings();
+
+            LootLockerSDKManager.StartGuestSession((response) =>
+            {
+                if (!response.success)
+                {
+                    Debug.Log("Could not start guest session");
+                    return;
+                }
+                FetchPlayerWallet(response.player_ulid);
+                FetchVirtualProductsFromCatalog();
+
+            });
+        }
+
+        public void FetchPlayerWallet(string ulid)
+        {
+            LootLockerSDKManager.GetWalletByHolderId(ulid, LootLockerEnums.LootLockerWalletHolderTypes.player, (response) =>
+            {
+                if (!response.success)
+                {
+                    Debug.Log("Could not start guest session");
```

Comment:
```suggestion
                    Debug.Log("Could not get wallet, error: \""+response.errorData.ToString()+"\"");
```
Not sure that this compiles, but I think we can teach users in our samples how to use errorData ;)

Reply:
Same here